### DISCUSSION TITLE
Optimizing images size

### DISCUSSION THREAD

**Discussion by avxkim** [strapi/strapi#6357] [Legacy Topics]
Is there a way to optimize images via imagemin, etc? Any guides?

**Comment by lauriejim** [comment]
Hello! If you want to do that, you will have to customize the code of the upload plugin.
Here is the [customization documentation](https://strapi.io/documentation/v3.x/concepts/customization.html)
And here is the [source code](https://github.com/strapi/strapi/tree/master/packages/strapi-plugin-upload) of the plugin.
So add your node_module and update the function to add your optimization code.
